content/altinn-studio/getting-started/app-dev-course/_index.nb.md

```diff
@@ -5,42 +5,42 @@ tags: [apps, build, deploy, training]
 ---
 
 {{% notice warning %}}
-Dette kurset er under oppgradering. Kun 5 moduler er foreløpig tilgjengelig på denne oppdaterte versjonen. 
+Dette kurset er under oppgradering. Kun seks moduler er foreløpig tilgjengelige på denne oppdaterte versjonen. 
 
 _Konsepter som dekkes i den oppdaterte versjonen:_
 - Lage tjeneste/app
 - Tilpasse datamodell
 - Lage skjema
 - Flere sider og informasjonsside
 - Publisering og testmiljø
+- Flervalgskomponenter og kodelister
 
 _Konsepter som foreløpig ikke dekkes i den oppdaterte versjonen:_
 - Dynamikk i skjema
-- Flervalgskomponenter og kodelister
 - Forhåndsutfylling (prefill)
 - Avansert logikk
 - Kall mot eksterne API
 - Egendefinert PDF visning
 
 Listene over oppdateres fortløpende når det legges til nye moduler.
-I mellomtiden ligger den gamle versjonen av kurset [her](../../../altinn-studio/getting-started/app-dev-course-old/) - flere av konseptene
+I mellomtiden ligger den gamle versjonen av kurset [her](../../../altinn-studio/getting-started/app-dev-course-old/) – flere av konseptene
 som ikke dekkes av det oppdaterte kurset dekkes der.
 {{% /notice %}}
 
-Introkurset er delt opp i 5 moduler og dekker et fiktivt case for Sogndal kommune.
+Introkurset er delt opp i seks moduler og dekker et fiktivt case for Sogndal kommune.
 Hver modul har en beskrivelse av oppgaven og lenker til relevant dokumentasjon.
 
 Du kan velge å utføre oppgavene i [Altinn Studio Designer](../) 
 (grafisk brukergrensesnitt med dra-og-slipp) eller i et [lokalt utviklingsmiljø](/nb/altinn-studio/guides/local-dev) 
 (krever noe teknisk kompetanse).
-Du kan også kombinere disse fremgangsmåtene. Alle de tilgjengelige modulene kan utfæres i Altinn Studio Designer.
+Du kan også kombinere disse fremgangsmåtene. Alle de tilgjengelige modulene kan utføres i Altinn Studio Designer.
 
 I løpet av disse modulene vil du ha vært innom den mest sentrale funksjonaliteten for en Altinn-app,
 samt testing av applikasjonen lokalt og i testmiljø.
 
 ## Før du starter
 
-Før du kan komme i gang må du [opprette en bruker](../create-user/) i Altinn Studio.
+Før du kan komme i gang må du [opprette en bruker i Altinn Studio](../create-user/).
 
 Les gjennom kravspesifikasjonene til Sogndal kommune og jobb deg gjennom modul for modul.
```

Comment:
Les gjennom oppgaven du skal utføre for Sogndal kommune og gå gjennom den steg for steg.

Reply:
Jeg tror vi bør skille mellom "kravspesifikasjoner" og "oppgaver" her. Kravspesifikasjoner er mer å anse som akseptansekriterier, mens oppgaver er instruksjonene leseren må utføre for å tilfredsstille dem. Den første siden er altså ikke en modul. Den inneholder ingen instruksjoner, bare ønskene til kommunen.

Reply:
Ja, jeg tenker bare at å kalle det "kravspesifikasjon" er litt heftig. "Kravene Sogndal kommune har til appen du skal lage" eller "ønskene"?

Reply:
Har byttet "kravspesifikasjon" med "ønske"